install-linux.sh

```diff
@@ -39,7 +39,7 @@ echo ""
 echo "Compile Axle-OS now?: Press r to continue..."
 read -n1 -rsp "" key
 if [ "$key" = "r" ]; then
-	make axle.iso CC=i686-toolchain/$arch/bin/i686-elf-gcc \
+	sudo make axle.iso CC=i686-toolchain/$arch/bin/i686-elf-gcc \
```

Comment:
Why does building require sudo?

Reply:
Probably just a left-over that I forgot about.
I will revert this change.